perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/MembersManagerMethod.java

```diff
@@ -812,6 +812,30 @@ public Member call(ApiCaller ac, Deserializer parms) throws PerunException {
 		}
 	},
 
+	/*#
+	 * Returns rich members by their IDs with specific attributes.
+	 *
+	 * @param ids List<Integer> list of members IDs
+	 * @param attrsNames List<String> Attribute names. If the list is null or empty, no attributes will be provided.
```

Comment:
If parameter `attrNames` is optional, we create docs for each version of the method - with and without the parameter.

Reply:
Fixed